packages/react-core/src/Authenticator/hooks/useAuthenticatorProps/constants.ts

```diff
@@ -0,0 +1,31 @@
+import { CommonRouteSelectorKeys, RouteSelectorKeys } from './types';
+
+const COMMON_SELECTOR_KEYS: CommonRouteSelectorKeys[] = [
```

Comment:
These are all related to input and form handling, which is what the commonality is, do you think we can be more specific here and call it `FORM_SELECTOR_KEYS`? Though I'm not sold on the selector bit, why selector?

Reply:
`error` and `isPending` are related to state machine "states" and are not directly tied to form handling, `updateBlur` and `updateForm` should be removed once validation is moved out of the state machine entirely. The state machine itself should have no knowledge of form or input events, but there is only so much tech debt that can be handled rn.

 Additionally,  `COMMON` scales for other keys that may be added in the future, for example if we were to add `user`. 

The usage of `SELECTOR` is because these are keys that we use for the selector param when calling `useAuthenticator`